composer.json

```
@@ -13,7 +13,8 @@
     "ext-zip": "*",
     "sentry/sdk": "^3.5",
     "smarty/smarty": "^5.5.1",
-    "google/apiclient": "^2.2.3"
+    "google/apiclient": "^2.2.3",
+    "composer require vlucas/phpdotenv": "^5.6.2"
```

This one needs to be changed, because its not valid json 😅 

```suggestion
    "vlucas/phpdotenv": "^5.6.2"
```